packages/core/package.json

```diff
@@ -2,13 +2,13 @@
   "name": "@web3-storage/content-claims",
   "version": "4.0.0",
   "description": "Implementation of the Content Claims Protocol.",
-  "main": "src/index.js",
+  "main": "./types/index.js",
```

Comment:
in this case should we rename types to dist? feels weird having the source in a folder called types...

Reply:
i'm open to it but i didn't want to change it from what it was and wanted to get ideas. I agree types is gross. dist is conventional and fine so it's a good option. I was thinking what if we start using `lib`.... but that's when I decided not to do it in this PR :P

Reply:
I like `dist` fwiw!